setup.py

```diff
@@ -0,0 +1,47 @@
+import os
+import re
+from setuptools import setup, find_packages
+
+PACKAGE_NAME = 'homeassistant'
+HERE = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(HERE, PACKAGE_NAME, 'const.py')) as fp:
+    VERSION = re.search("__version__ = ['\"]([^']+)['\"]\n", fp.read()).group(1)
+
+PACKAGES = find_packages() + \
+    ['homeassistant.external', 'homeassistant.external.noop',
+     'homeassistant.external.nzbclients', 'homeassistant.external.vera']
+
+PACKAGE_DATA = \
+    {'homeassistant.components.frontend': ['index.html.template'],
+     'homeassistant.components.frontend.www_static': ['*.*'],
+     'homeassistant.components.frontend.www_static.images': ['*.*']}
+
+setup(
+    name=PACKAGE_NAME,
+    version=VERSION,
+    license='MIT License',
+    url='https://home-assistant.io/',
+    download_url='https://github.com/automicus/pyisy/tarball/0.7.0',
+    author='Paulus Schoutsen',
+    author_email='paulus@paulusschoutsen.nl',
+    description='Open-source home automation platform running on Python 3.',
+    packages=PACKAGES,
+    include_package_data=True,
+    package_data=PACKAGE_DATA,
+    zip_safe=False,
+    platforms='any',
+    install_requires=['requests==2.7.0', 'pyyaml==3.11', 'pytz==2015.4'],
```

Comment:
Maybe we can be a bit more flexible here. Like `requests>2,<3`, `pyyaml>3.11,<4`, `pytz>=2015.4`

For pytz I am not afraid that it will add breaking changes in future releases